Tools/boringNetworkTool/main.py

```diff
@@ -0,0 +1,83 @@
+import socket
+import sys
+import time
+import json
+
+
+class Message:
+    def __init__(self, msgID, isExtendedID, data):
+        self.msgID = msgID
+        self.isExtendedID = isExtendedID
+        self.data = data
+    def toString(self):
+        return "" + str(self.msgID) + "\n" + str(self.isExtendedID) + "\n" + str(self.data)
+
+def createMessages(raw_messages):
+    msgs = []
+    for message in raw_messages:
+        msgs.append(Message(msgID=message["msgId"],
+                            isExtendedID=message["isExtendedId"],
+                            data=message["data"]
+                            ))
+    return msgs
+
+
+def sendWithUDP(msg_config, host, port):
+    try:
+        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    except socket.error:
+        print('Failed to create socket')
+        sys.exit()
+    msgs = createMessages(msg_config["config"]["messages"])
+    msg_frequency = msg_config["config"]["messageFrequency"]
+    for message in msgs:
+        time.sleep(1.0 / msg_frequency)
+        try:
+            # Set the whole string
+            s.sendto(bytes(message.toString(), 'utf-8'), (host, port))
+            print("message sent:" + message.toString())
+            # receive data from client (data, addr)
+            d = s.recvfrom(1024)
```

Comment:
This blocks, with UDP I don't think we will always want to wait for a reply from the server before sending the next packet. Potentially, we could have an option in the config to await a reply.

Where it is a debug tool, we will usually just want it to send data continuously as well. This could be another option in the config (bool sendContinuous), that if true the script just continuously calls sendUDP or sendTCP. 

I tried a while loop, it worked fine for UDP but there was an error when calling send TCP in a while loop (the server reset the connection or something)

![image](https://user-images.githubusercontent.com/47944858/111917740-8ead0b00-8a64-11eb-928e-49cb4e86d323.png)